iocore/net/quic/test/test_QUICHandshakeProtocol.cc

```diff
@@ -405,13 +376,11 @@ TEST_CASE("QUICHandshakeProtocol")
     msg1.offsets[3]  = msg1_len;
     msg1.offsets[4]  = msg1_len;
 
-    QUICHandshakeMsgs msg2;
-    uint8_t msg2_buf[MAX_HANDSHAKE_MSG_LEN] = {0};
-    msg2.buf                                = msg2_buf;
-    msg2.max_buf_len                        = MAX_HANDSHAKE_MSG_LEN;
-
+    QUICHandshakeMsgs *msg2 = nullptr;
     CHECK(server->handshake(&msg2, &msg1) != 1);
-    CHECK(msg2.error_code == 0x10a); //< 0x100 + unexpected_message(10)
+    // This doesn't pass with BoringSSL (feature removed)
```

Comment:
I removed code that sets error_code because it requires extra code to parse SSL messages. With the hacked OpenSSL we have to read SSL message one by one anyway, but not with BoringSSL API.

I'm going to remove this test.

Reply:
Where/How is the TLS alerts handled? IIRC, TLS alerts should be turned into a QUIC error code.

Reply:
OK, that is probably something has to be done when `send_alert` callback function is called. It's blank now :p

I'm not sure if we can do the same thing by using this msg structure with BoringSSL, but I understood the need.